students/gotem888/hw-12 searchEngine/index.html

```diff
@@ -0,0 +1,47 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+        <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+        <meta charset = "utf-8">
+        <link rel="stylesheet" href="./css/reset.css">
+        <link rel="stylesheet" href="./css/style.css">
+        <link rel="preconnect" href="https://fonts.googleapis.com">
+        <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+        <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@300&family=Roboto:wght@300&display=swap" rel="stylesheet">
+        <title>HW - SearchEngine</title>
+    </head>
+    <body>
+        <div class="wrapper">
+            <aside class="wrapper__filters">
+                <div class="posCalibrate">
+                <form name="filtersForm" class="form" >
+                    <div class="form__item inputArea">
```

Comment:
To be consistent name classes with [BEM approach](https://ru.bem.info/methodology/).

And don't forget to rename it styles as well.
```suggestion
                    <div class="form__item form__item--text">
```